### ISSUE TITLE
SIGWINCH ignored. Not daemonized

### ISSUE THREAD

**Issue by pierreNomazy** [benoitc/gunicorn#449]
Hello,

I tried gunicorn with a django chat apps. Chat is working fine with run server, but with gunicorn, I get error message every 2 seconds :
Handling signal: winch
SIGWINCH ignored. Not daemonized

**Comment by sarkeribrahim** [comment]
anyone knows this type of problem?

2015-08-25 16:34:22 [20150825-163104-971b] [ERROR] Train Caffe Model: Check failed: *ptr host allocation of size 116160000 failed
2015-08-25 16:34:28 [20150825-163104-971b] [ERROR] Train Caffe Model task failed with error code -6

Job Status Error

Initialized at 04:31:04 PM (1 second)
Running at 04:31:05 PM (3 minutes, 25 seconds)
Error at 04:34:31 PM
(Total - 3 minutes, 26 seconds)

Train Caffe Model Error

Initialized at 04:31:04 PM (1 second)
Running at 04:31:06 PM (3 minutes, 23 seconds)
Error at 04:34:29 PM
(Total - 3 minutes, 24 seconds)

ERROR: Check failed: *ptr host allocation of size 116160000 failed

conv2 needs backward computation.
pool1 needs backward computation.
norm1 needs backward computation.
relu1 needs backward computation.
conv1 needs backward computation.
label_data_1_split does not need backward computation.
data does not need backward computation.
This network produces output accuracy
This network produces output loss
Collecting Learning Rate and Weight Decay.
Network initialization done.
Memory required for data: 831529208
Solver scaffolding done.
Starting Optimization
Solving
Learning Rate Policy: step
Iteration 0, Testing net (#0)
Test net output #0: accuracy = 0.34
Test net output #1: loss = 1.09667 (* 1 = 1.09667 loss)
Check failed: *ptr host allocation of size 116160000 failed

**Comment by tilgovi** [comment]
@sarkeribrahim that looks like a problem with your application and nothing to do with gunicorn. Please only comment on closed issues if you think you have the same problem.
